docs/config-file-ort-yml.md

```diff
@@ -156,6 +156,25 @@ For details of the specification, see
 The list of available options for `reason` are defined in
 [LicenseFindingCurationReason.kt](../model/src/main/kotlin/config/LicenseFindingCurationReason.kt).
 
+### Curating Third-Party Metadata
```

Comment:
Curations are not limited to third-Party packages but could also curated your orgs' packages.

Reply:
Removed `Third-Party`.